server/src/main/java/com/linecorp/centraldogma/server/storage/StorageManager.java

```diff
@@ -96,23 +96,16 @@ default T create(String name, Author author) {
     T unremove(String name);
 
     /**
-     * Returns a set of names for the elements which have been purged.
+     * Purges a set of names for the elements which have been marked for purging.
      */
-    Set<String> listPurged();
+    void purgeMarked();
 
     /**
-     * Purges an element with the specified {@code name}.
+     * Marks for purging an element with the specified {@code name}.
      *
      * @param name the name of an element which is supposed to be purged
      */
-    void purge(String name);
-
-    /**
-     * Clears an element permanently with the specified {@code name}.
-     *
-     * @param name the name of an element which is supposed to be cleared
-     */
-    void clear(String name);
+    void markForPurge(String name);
```

Comment:
I think we need to be consistent with the documentation: `markForPurging` or `markForPurge`.

Reply:
I chose `markForPurge` because it is shorter and simple. 😁
For synchronizing with that, the documentation was updated.